src/stylesheets/components/_box-one.scss

```diff
@@ -57,6 +57,10 @@
     transform: translateX(-45%) translateY(-18%);
     width: 9em;
   }
+
+  @media ($screen-xs-up) and ($screen-sm) {
```

Comment:
i'd say you are probably looking at your breakpoints backwards. i've found the best way to use them is start at 0px width and move up. so the first styles you define should be catered toward the smallest possible breakpoint and then once it starts to look bad (at say, 400px) you add or modify your properties, and keep going this way until it's done. 

when you start defining specific breakpoints that start at a minimum and stop at a maximum, you are making it difficult to inherit, therefore increasing the probability that you will be repeating code. on top of this, it creates CSS that is hard to read for others, having to keep where your arbitrary start and end widths in their heads. lastly, it's really difficult to refactor when you start defining arbitrary breakpoint windows.

so it should look something like this:

```css
.pseudo-code
  /* default/min-width properties here */
  background-color: red;
  height: 100px;
  width: 100%;

  @media (min-width: 400px) {
    height: 200px;
  }
  @media (min-width: 768px) {
    height: auto;
    width: 50%;
  }
/* and so on */
}
```
you really should not be setting a max-width in the breakpoint unless there is really a need for it, otherwise, you should be changing the property in the next breakpoint up.